### DISCUSSION TITLE
LRT Plius [LT]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#400] [Show and tell]
### Channel ID

LRTPlius.lt

### Stream URL

https://plius.lrt.lt/secure/plius/hls/8VYboTVWtyIXaUxktfatoA/1670759126/lrt-plius.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
404ed - geo-blocked?

**Comment by freearhey** [comment]
https://plius.lrt.lt/secure/plius/hls/8VYboTVWtyIXaUxktfatoA/1670759126/lrt-plius.m3u8 (offline)
<img width="600" alt="image" src="https://user-images.githubusercontent.com/7253922/232247751-d448ea0c-3000-40ba-8c4f-e78ee2c0698c.png">

**Comment by DJ1999-max** [comment]
Channel only, fully in LT, but some progammes are seen in website.
